.editorconfig

```diff
@@ -1,8 +1,12 @@
-# EditorConfig is awesome: https://EditorConfig.org
-
-# top-most EditorConfig file
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
 root = true
 
+# C# files
+[*.cs]
```

Comment:
This needs to be moved below the default `[*]` section, e.g. line 17/18 currently.

Otherwise the csharp and dotnet rules will propably apply to all files instead of only `*.cs` files